learn/advanced/faceted_search.md

```diff
@@ -0,0 +1,215 @@
+      "max":4.7
+    }
+  }
+  …
+}  
+```
+
+If none of the matching documents have a numeric value for a facet, that facet is not included in the `facetStats` object. Since `rating` was the only numeric facet in our example, it is the only facet returned in the `facetStats` object.
+
+## Facet types
+
+### Conjunctive facets
+
+Conjunctive facets use the `AND` logical operator. When users select multiple values for a facet, returned results must contain all selected facet values.
+
+With conjunctive facets, when a user selects `English` from the `language` facet, all returned books must be in English. If the user further narrows down the search by selecting `Fiction` and `Literature` as `genres`, all returned books must be in English and contain both `genres`.
+
+```
+"language = English AND genres = Fiction AND genres = Literature"
+```
+
+The GIF below shows how the facet count for `genres` updates to only include books that meet **all three conditions**.
+
+![Selecting English books with 'Fiction' and 'Literature' as 'genres' for the books dataset](/faceted-search/conjunctive-factes.gif)
+
+### Disjunctive facets
+
+Disjunctive facets use the `OR` logical operator. When users select multiple values for a facet, returned results must contain at least one of the selected values.
+
+With disjunctive facets, when a user selects `Fiction`, and `Literature`, Meilisearch returns all books that are either `Fiction`, `Literature`, or both:
+
+```
+"genres = Fiction OR genres = Literature"
+```
+
+The GIF below shows the `books` dataset with disjunctive facets. Notice how the facet count for `genres` updates based on the selection.
+
+![Selecting 'Fiction' and 'Literature' as 'genres' for the books dataset](/faceted-search/disjunctive_facets.gif)
+
+### Combining conjunctive and disjunctive facets
+
+It is possible to create search queries with both conjunctive and disjunctive facets.
+
+For example, a user might select `English` and `French` from the `language` facet so they can see books written either in English or in French. This query uses an `OR` operator and is a disjunctive facet:
+
+```
+"language = English OR language = French"
+```
+
+The same user might also be interested in literary fiction books and select `Fiction` and `Literature` as `genres`. Since the user wants a specific combination of genres, their query uses an `AND` operator:
+
+```
+"genres = Fiction AND genres = Literature"
+```
+
+The user can combine these two filter expressions in one by wrapping them in parentheses and using an `AND` operator:
+
+```
+"(language = English OR language = French) AND (genres = Fiction AND genres = Literature)"
+```
+
+The GIF below shows the `books` dataset with conjunctive and disjunctive facets. Notice how the facet count for each facet updates based on the selection.
+
+![Selecting 'Fiction' and 'Literature' as 'genres' for English books](/faceted-search/conjunctive-and-disjunctive-facets.gif)
+
+::: tip Improving user experience
```

Comment:
As a reader of the documentation, I don't really understand how using multi search helps improving user experience, is this related to the "cool behavior"?

I see this was discussed in https://github.com/meilisearch/documentation/pull/2208#discussion_r1150909298 already, with @guimachiavelli asking we remove the tip. I think achieving the "cool behavior" is rather integral to faceted search for e-commerce sites, so I'd rather we keep this even in a limited form.

On the other hand we could also leave this tip out entirely and revisit this documentation to add a link to a full-fledged blogpost about faceted search for e-commerce (that would include an explanation of the "cool behavior") once such a blogpost is ready.

Reply:
It is related to the "cool behavior". Initially, the tip was too detailed, and we felt it was out of scope of this guide. [See comment](https://github.com/meilisearch/documentation/pull/2208#discussion_r1151947741)
Let's go with removing the tip for now, then. We can revisit after and add a more detailed explanation or link to a blog or tutorial (when we have one).